proof/invariant-abstract/DetSchedSchedule_AI.thy

```diff
@@ -9408,6 +9423,14 @@ lemma handle_overrun_loop_length:
            | wps)+
   done
 
+lemma handle_overrun_loop_body_sc_at_cur_sc[wp]:
```

Comment:
I think this name suggests something different from what is stated there, but it probably should be actually about `sc_at` or something equivalent. See the comment above.